chains/mainnet/kujira.js

```diff
@@ -29,8 +29,8 @@ module.exports = {
     icsFromTerra: {
       contract:
         'terra1e0mrzy8077druuu42vs0hu7ugguade0cj65dgtauyaw4gsl4kv0qtdf2au',

```

Comment:
Looks like the ICS contract used for this channel is `terra1d90p5lacfxnqgjxjupu234lxnxyeu8fdeef4d0e0nqy3p30r7gss4myn9x`

https://phoenix-lcd.terra.dev/ibc/core/channel/v1/channels/channel-34/ports/wasm.terra1d90p5lacfxnqgjxjupu234lxnxyeu8fdeef4d0e0nqy3p30r7gss4myn9x

Reply:
We also have to update the new IBC objects that are coming with this PR here: https://github.com/terra-money/station-assets/pull/161/files